templates/core/about/index.html

```diff
@@ -25,18 +25,18 @@ <h1 id="crate-title">About Docs.rs</h1>
         </p>
 
 
-        <h2 id="more-about-docsrs">More about Docs.rs</h2>
+	 <h2 id="more-about-docsrs"> <a href="#more-about-docsrs">More about Docs.rs</a> </h2>
```

Comment:
I don't think you meant to change the spacing here, please keep the existing spaces (not tabs).
```suggestion
        <h2 id="more-about-docsrs"> <a href="#more-about-docsrs">More about Docs.rs</a> </h2>
```